doonut-app-external-api/src/main/kotlin/com/doonutmate/calendar/controller/CalendarController.kt

```diff
@@ -0,0 +1,29 @@
+package com.doonutmate.calendar.controller
+
+import com.doonutmate.calendar.controller.dto.CalendarResponse
+import com.doonutmate.calendar.controller.dto.CalendarResult
+import com.doonutmate.calendar.service.CalendarService
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.tags.Tag
+import org.springframework.data.domain.PageRequest
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import java.time.Instant
+
+@RestController
+@Tag(name = "Calendar", description = "캘린더 API")
+@RequestMapping("/calendar")
```

Comment:
REST API 설계 원칙에 따르면 url은 단수보다는 복수형을 추천하고 있어요.  
이미 설계한 API는 그대로 가더라도 이제부터 복수형으로 컨벤션을 가져가는건 어떨까요?

```suggestion
@RequestMapping("/calendars")
```